changelog/6273.improvement.md

```diff
@@ -0,0 +1 @@
+Add `output_channel` query param to `/conversations/<conversation_id>/tracker/events` route. Forward bot messages received by route to output channel.
```

Comment:
We should mention `execute_side_effects` here as well, and how it works in combination with `output_channel`.

Reply:
@federicotdn Thanks! Here I updated the changelog.